platform/broadcom/rules.dep

```diff
@@ -17,6 +17,7 @@ include $(PLATFORM_PATH)/platform-modules-brcm-xlr-gts.dep
 include $(PLATFORM_PATH)/docker-syncd-brcm.dep
 include $(PLATFORM_PATH)/docker-syncd-brcm-rpc.dep
 include $(PLATFORM_PATH)/docker-saiserver-brcm.dep
+include $(PLATFORM_PATH)/docker-gbsyncd-credo.dep
```

Comment:
credo is not specific to broadcom, i feel this docker should be a generic docker except on vs platform or we can say which platform will include this docker initially it would be on broadcom only, but this docker can be available on other platforms as well.